replibyte/src/cli.rs

```diff
@@ -5,7 +5,6 @@ use clap::{Args, Parser, Subcommand};
 /// Replibyte is a tool to seed your databases with your production data while keeping sensitive data safe, just pass `-h`
 #[derive(Parser, Debug)]
 #[clap(about, long_about = None)]
-#[clap(propagate_version = true)]
```

Comment:
@evoxmusic I've remove this as there is no version attribute, this lead to this error:

```sh
thread 'main' panicked at 'Command replibyte: No version information via Command::version or Command::long_version to propagate'
```

As we discussed in Discord at some point, to put back the version we must first handle it properly.

Reply:
Perfect - sounds good to me. Thx Fab